packages/jest/README.md

```diff
@@ -12,6 +12,7 @@ The builder comes to provide zero configuration setup for Jest while keeping the
 <details>
   <summary>Click to expand</summary>
 
+- [Version 20](https://github.com/just-jeb/angular-builders/blob/20.x.x/packages/jest/README.md)
```

Comment:
Also add Version 21?

Reply:
These are previous versions, version 21 will be the current version